save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -53,7 +49,6 @@ class DemoManagerController(
 
     @PostMapping("/{organizationName}/{projectName}/add")
     @RequiresAuthorizationSourceHeader
-    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
```

Comment:
why do you remove it?

Reply:
It was because all the save-demo was in development. Not it is time to fix bugs so I decided to remove it